src/plugins/intel_cpu/src/nodes/mkldnn_fullyconnected_node.cpp

```diff
@@ -128,64 +198,177 @@ void MKLDNNFullyConnectedNode::getSupportedDescriptors() {
     }
 }
 
-void MKLDNNFullyConnectedNode::createPrimitive() {
-    if (prim)
-        return;
+void MKLDNNFullyConnectedNode::prepareParams() {
+    auto srcMemPtr = getParentEdgesAtPort(0)[0]->getMemoryPtr();
+    auto wghMemPtr = getParentEdgesAtPort(1)[0]->getMemoryPtr();
+    auto dstMemPtr = getChildEdgesAtPort(0)[0]->getMemoryPtr();
+    if (!dstMemPtr || !dstMemPtr->GetPrimitivePtr())
+        IE_THROW() << "Destination memory hasn't been allocated.";
+    if (!srcMemPtr || !srcMemPtr->GetPrimitivePtr())
+        IE_THROW() << "Input memory hasn't been allocated.";
+    if (!wghMemPtr || !wghMemPtr->GetPrimitivePtr())
+        IE_THROW() << "Weight memory hasn't been allocated.";
+    MKLDNNMemoryPtr biasMemPtr = nullptr;
+    if (withBiases) {
+        biasMemPtr = getParentEdgesAtPort(2)[0]->getMemoryPtr();
+        if (!biasMemPtr || !biasMemPtr->GetPrimitivePtr())
+            IE_THROW() << "Input memory hasn't been allocated.";
+    }
+
+    const NodeDesc *selected_pd = getSelectedPrimitiveDescriptor();
+    if (selected_pd == nullptr)
+        IE_THROW() << "Preferable primitive descriptor is not set for node " << getName() << ".";
+
+    auto initPrimitiveAttr = [&]() {
```

Comment:
This lambda has the same name as FC node method and also called just once. I would suggest to remove lambda at all.

Reply:
fixed in: https://github.com/openvinotoolkit/openvino/pull/9621